### ISSUE TITLE
Unforking? What about RehabMan changes?

### ISSUE THREAD

**Issue by timofonic** [al3xtjames/Clover#1]
Hello. @al3xtjames 

Thanks a lot for your efforts!

Are there any unforking reposibilities? What about @RehabMan fork?

Kind regards from Spain... ;)

**Comment by al3xtjames** [comment]
Hi,

What do you mean by unforking? Are you asking if I plan on getting my changes merged into the SVN repo? If so, the answer is eventually. Some of the changes I've made (such as the updated Intel HD Graphics/HDA device property injectors) are definitely incomplete, and need some more work before they could fully replace the original implementations in Clover. Some of the changes in this repo have already been merged by Slice (like disabling Ethernet `device_type` injection).

As for RehabMan's fork, I believe most (all?) of it was merged by Slice recently, so I just need to rebase to the latest SVN revision to add his changes to my fork.

**Comment by timofonic** [comment]
@al3xtjames  Sorry, I wrote a reply but had hardware issues and lost all the message. Then I got busy.

Thanks a lot for your explanation, it was really interesting.

Why not join and work in a common Clover branch? Why does Clover still uses SourceForge's SVN and their GitHub mirror is obsolete? :O 

I get confused about the dates in GitHub, it says latest commit was 10 days ago or so but there's a commit about 4 days ago if you look at the main page. Why?

I'm not sure if you looked at it, but latest TravisCI build log complains about certain issues and fails to build. Here are some excerpts of the log that I found relevant, sorry if I didn't choose the most really relevant ones:

Fix WWDRCA Certificate
Unable to delete certificate matching "0950B6CD3D2F37EA246A1AAA20DFAADBD6FE1F75"security: AppleWWDRCA.cer: already in /Library/Keychains/System.keychain


antlr.c:81:18: warning: equality comparison with extraneous parentheses [-Wparentheses-equality]  (it happens tons of times over the code. False positive or what?)

antlr.c:3255:118: warning: expression result unused [-Wunused-value]

30 warnings generated.
138 warnings generated.
PcdValueCommon.c:270:47: error: format specifies type 'unsigned long' but the argument has type 'UINT64' (aka 'unsigned long long') [-Werror,-Wformat]
    sprintf(PcdList[Index].Value, "0x%016lx", Value);
                                     ~~~~~~   ^~~~~
                                     %016llx
/usr/include/secure/_stdio.h:47:56: note: expanded from macro 'sprintf'
  __builtin___sprintf_chk (str, 0, __darwin_obsz(str), __VA_ARGS__)
                                                       ^~~~~~~~~~~
1 error generated.
make[2]: *** [PcdValueCommon.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [Common] Error 2
make[1]: *** Waiting for unfinished jobs....
Antlr parser generator   Version 1.33MR33   1989-2001
VfrSyntax.g, line 2018: warning: predicate: LT(i) missing, bad, or with i=0; assuming i=1
VfrSyntax.g, line 2023: warning: predicate: LT(i) missing, bad, or with i=0; assuming i=1
VfrSyntax.g, line 3651: warning: alts 1 and 2 of {..} ambiguous upon ( ";" RefreshGuid GuidOp Locked Image EndIf InconsistentIf DisableIf SuppressIf Default GrayOutIf )
VfrSyntax.g, line 3660: warning: alts 1 and 2 of {..} ambiguous upon ( ";" RefreshGuid GuidOp Locked Image EndIf InconsistentIf DisableIf SuppressIf Default GrayOutIf )
VfrSyntax.g, line 3669: warning: alts 1 and 2 of {..} ambiguous upon ( ";" RefreshGuid GuidOp Locked Image EndIf InconsistentIf DisableIf SuppressIf Default GrayOutIf )
VfrSyntax.g, line 3679: warning: alts 1 and 2 of {..} ambiguous upon ( ";" RefreshGuid GuidOp Locked Image EndIf InconsistentIf DisableIf SuppressIf Default GrayOutIf )
VfrSyntax.g, line 3709: warning: alts 1 and 2 of {..} ambiguous upon ( ";" RefreshGuid GuidOp Locked Image EndIf InconsistentIf DisableIf SuppressIf Default GrayOutIf )
VfrSyntax.g, line 3718: warning: alts 1 and 2 of {..} ambiguous upon ( ";" RefreshGuid GuidOp Locked Image EndIf InconsistentIf DisableIf SuppressIf Default GrayOutIf )
dlg  Version 1.33MR33   1989-2001
make: *** [Source/C] Error 2
The command "make -C BaseTools -s -j 6" failed and exited with 2 during .
Your build has been stopped.

PS: It seems you are building using a Mac VM and LLVM. What about testing the code against latest GCC versions under Linux too? :D
